dspace-server-webapp/src/test/java/org/dspace/app/rest/GroupRestRepositoryIT.java

```diff
@@ -761,259 +754,136 @@ public void addChildGroupTest() throws Exception {
-            childGroup2 = groupService.create(context);
-
-            groupService.addMember(context, childGroup1, parentGroup);
-            groupService.update(context, childGroup1);
-
-            context.commit();
-
-            parentGroup = context.reloadEntity(parentGroup);
-            childGroup1 = context.reloadEntity(childGroup1);
-            childGroup2 = context.reloadEntity(childGroup2);
+        context.turnOffAuthorisationSystem();
 
-            context.restoreAuthSystemState();
-            String authToken = getAuthToken(admin.getEmail(), password);
+        Group parentGroup = GroupBuilder.createGroup(context).build();
+        Group childGroup1 = GroupBuilder.createGroup(context)
+                .withParent(parentGroup)
+                .build();
+        Group childGroup2 = GroupBuilder.createGroup(context).build();
 
-            getClient(authToken).perform(
-                    post("/api/eperson/groups/" + parentGroup.getID() + "/subgroups")
-                            .contentType(parseMediaType(TEXT_URI_LIST_VALUE))
-                            .content(REST_SERVER_URL + "eperson/groups/123456789\n"
-                                    + REST_SERVER_URL + "eperson/groups/" + childGroup2.getID()
-                            )
-            ).andExpect(status().isUnprocessableEntity());
+        context.restoreAuthSystemState();
+        String authToken = getAuthToken(admin.getEmail(), password);
 
-            getClient(authToken).perform(
-                    post("/api/eperson/groups/" + parentGroup.getID() + "/subgroups")
-                            .contentType(parseMediaType(TEXT_URI_LIST_VALUE))
-                            .content(REST_SERVER_URL + "eperson/groups/" + childGroup1.getID() + "/\n"
-                                    + REST_SERVER_URL + "eperson/groups/" + childGroup2.getID()
-                            )
-            ).andExpect(status().isUnprocessableEntity());
+        getClient(authToken).perform(
+                post("/api/eperson/groups/" + parentGroup.getID() + "/subgroups")
+                        .contentType(parseMediaType(TEXT_URI_LIST_VALUE))
+                        .content(REST_SERVER_URL + "eperson/groups/123456789\n"
+                                + REST_SERVER_URL + "eperson/groups/" + childGroup2.getID()
+                        )
+        ).andExpect(status().isUnprocessableEntity());
 
-        } finally {
-            if (parentGroup != null) {
-                GroupBuilder.deleteGroup(parentGroup.getID());
-            }
-            if (childGroup1 != null) {
-                GroupBuilder.deleteGroup(childGroup1.getID());
-            }
-            if (childGroup2 != null) {
-                GroupBuilder.deleteGroup(childGroup2.getID());
-            }
-        }
+        // TODO - confirm with reviewers that this is a mistake - it actually should be No Content
+        //        (see AddMember test) but was incorrectly expecting 422?
```

Comment:
I think you are correct in this TODO.  You found a bug in the test and this test should return No Content.  So, this TODO can likely be removed.